### PR TITLE
Works and about section

### PR DESCRIPTION
**Pull request by Cisco-the-wayword**
- Added works section
- Added about section
- Implemented button
- Ran and passed linter tests
- Tested the website to see how well it works

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,45 +2,265 @@
...
+          <div class="css">
+            <p class="font">css</p>
+          </div>
+          <div class="js">
+            <p class="font">JavScript</p>
+          </div>
+          <div class="html">
+            <p class="font">html</p>
+          </div>
+        </div>
```
Comment (zieeco) [inline index.html]: - Please, kindly use the ul tag and add its list items as this is are list of skills, using the ul will enable you to enforce the use of html5 semantic tag correctly

File: index.html [hunk 2]
```diff
@@ -2,45 +2,265 @@
...
```
Comment (zieeco) [inline index.html]: - Please, kindly use the ul tag and add its list items as this is are list of skills, using the ul will enable you to enforce the use of html5 semantic tag correctly

File: index.html [hunk 3]
```diff
@@ -2,45 +2,265 @@
...
+          </div>
+          <div class="css">
+            <p class="font">css</p>
+          </div>
+          <div class="js">
+            <p class="font">JavScript</p>
+          </div>
+          <div class="html">
+            <p class="font">html</p>
+          </div>
```
Comment (zieeco) [inline index.html]: - Please, kindly use the ul tag and add its list items as this is are list of skills, using the ul will enable you to enforce the use of html5 semantic tag correctly

File: index.html [hunk 4]
```diff
@@ -2,45 +2,265 @@
...
+        </div>
```
Comment (zieeco) [inline index.html]: - Please, kindly use the ul tag and add its list items as this is are list of skills, using the ul will enable you to enforce the use of html5 semantic tag correctly

File: index.html [hunk 5]
```diff
@@ -2,45 +2,265 @@
...
```
Comment (zieeco) [inline index.html]: - Please, kindly use the ul tag and add its list items as this is are list of skills, using the ul will enable you to enforce the use of html5 semantic tag correctly

File: index.html [hunk 6]
```diff
@@ -2,45 +2,265 @@
...
+        <div class="button-space">
+          <button type="button" class="project-text">See Project
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="block">
+      <div class="gray-area"></div>
+      <div class="description">
+        <div class="post">Multi-Post Stories Gain+Glory</div>
```
Comment (zieeco) [inline index.html]: - Please, consider using an h2-h6 tag here, as this is the header and not a div item.

File: index.html [hunk 7]
```diff
@@ -2,45 +2,265 @@
...
```
Comment (zieeco) [inline index.html]: - Please, consider using an h2-h6 tag here, as this is the header and not a div item.

File: index.html [hunk 8]
```diff
@@ -2,45 +2,265 @@
...
```
Comment (zieeco) [inline index.html]: - Please, consider using an h2-h6 tag here, as this is the header and not a div item.

File: index.html [hunk 9]
```diff
@@ -2,45 +2,265 @@
...
+    </section>
+  </main>
+  <section class="rest">
+    <h1 class="works">My Recent Works</h1>
+    <div class="indicator"> <img id="line" src="Images/Indicator.png" alt="fancyLine"></div>
+    <div class="more-space"></div>
+    <div class="block">
+      <div class="gray-area"></div>
+      <div class="description">
+        <div class="post">Multi-Post Stories Gain+Glory</div>
```
Comment (zieeco) [inline index.html]: - Please, consider using an h2-h6 tag here, as this is the header and not a div item.

File: index.html [hunk 10]
```diff
@@ -2,45 +2,265 @@
...
+          </div>
+          <div class="css">
+            <p class="font">css</p>
+          </div>
+          <div class="js">
+            <p class="font">JavScript</p>
+          </div>
+          <div class="html">
+            <p class="font">html</p>
+          </div>
```
Comment (zieeco) [inline index.html]: - Please, kindly use the ul tag and add its list items as this is are list of skills, using the ul will enable you to enforce the use of html5 semantic tag correctly

File: index.html [hunk 11]
```diff
@@ -2,45 +2,265 @@
...
+        <div class="button-space">
+          <button type="button" class="project-text">See Project
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="block">
+      <div class="gray-area"></div>
+      <div class="description">
+        <div class="post">Multi-Post Stories Gain+Glory</div>
```
Comment (zieeco) [inline index.html]: - Please, consider using an h2-h6 tag here, as this is the header and not a div item.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Cisco-the-wayword,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 12]
```diff
@@ -111,9 +118,432 @@ nav {
...
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 305px;
+  height: 32px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: auto;
+}
```
Comment (mukhammadrizooff) [inline styles.css]: - [ ] [REQUIRED] Good job so far! Please fix the style error highlighted in the image below!

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Cisco-the-wayword, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @Cisco-the-wayword, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Perfect design ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**